robottelo/ui/locators/base.py

```diff
@@ -1687,8 +1687,9 @@
         By.XPATH, "//td/a[contains(., '%s')]/following::td/div/"
                   "button[contains(@class, 'dropdown-toggle')]"),
     "contentviews.remove_ver": (
-        By.XPATH, ("//td/a[contains(., '%s')]/following::td//"
-                   "a[contains(@ng-click, 'version-deletion')]")),
+        By.XPATH,
+        ("//td/a[contains(., '%s')]/following::td[@class='col-sm-2'][1]"
```

Comment:
I guess i'm missing something. Are we talking about the same page, CV versions list? On that page it works ok with multiple versions present, as there's no other options in a row with specific version - we're passing full version name to locator, so we select correct row for sure, different versions are in different rows. Or what's the case?